### DISCUSSION TITLE
Translation Approach for tr_organizedPageTitle in Portuguese

### DISCUSSION THREAD

**Discussion by fschein** [sws2apps/organized-app#2977] [Ideas]
We are currently discussing the best translation approach for the string tr_organizedPageTitle in the pt-BR, version of the app. The string seems to require translating the word Organized into an adjective that aligns with the context of the application. In pt-BR, a direct translation would be Organizado.

However, to maintain consistency across the app and preserve the branding, we believe it might be better to keep the app name, Organized, as it is. One suggestion is to adapt the phrase in pt-BR into something like:

>     Organize suas atividades com Organized

It would be something like this:

>     Organize your activities with Organized.

This approach ensures that the app's identity remains clear, while also conveying the intended meaning naturally in pt-BR.

We'd love to hear your thoughts on this matter. Should we prioritize translating the word into a local equivalent, or is it acceptable to keep the app name in English for branding consistency?

Looking forward to your feedback :)

**Comment by ux-git** [comment]
Thank you for your feedback! As noted in the Crowdin description, this case is unique since it highlights the congregation becoming more "organized" through the app, and therefore it's the only case when this word should actually be translated.

<img width="697" alt="Screenshot 2024-11-19 at 21 01 37" src="https://github.com/user-attachments/assets/fe6ac7aa-2b1b-4efc-a9df-0addeb1dc06d">


While we typically don’t translate "Organized" to maintain the brand name, here we need a creative approach:

1. Try to keep "organized" as an adjective, preferably phonetically similar to an English word, if possible. Thankfully the word "organized" sounds similar in many languages.

2. Structure the sentence to keep "organized" at the end, since the start of the sentence and the last word are two different translation keys.

While it’s sometimes straightforward, other times language differences make it tricky. In such cases, we aim for the best option, including using a local equivalent that fits naturally. Thanks again for the valuable input and hope that helps!

**Reply by fschein** [reply]
Got it! Thank you for the response :smile:
